providers/fips.module.sources

```diff
@@ -348,6 +356,7 @@ crypto/sha/asm/sha512-ia64.pl
 crypto/sha/asm/sha512-mips.pl
 crypto/sha/asm/sha512-parisc.pl
 crypto/sha/asm/sha512-ppc.pl
+crypto/sha/asm/sha512-riscv64-zvkb-zvknhb.pl
```

Comment:
what triggered the risc code getting included here?  The risc asm was added well over a year ago